bitnami/flux/values.yaml

```diff
@@ -664,6 +667,9 @@ helmController:
   ## @param helmController.args Override default container args (useful when using custom images)
   ##
   args: []
+  ## @param helmController.automountServiceAccountToken Mount Service Account token in pod
+  ##
+  automountServiceAccountToken: true
```

Comment:
shouldn't it be set to `false` instead?

Reply:
They require access to the K8s API so they need to mount the SA token